Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN cd /tmp && \
     pip3 install .
 
 # TODO: switch to proper release
-RUN pip3 install https://github.com/chanzuckerberg/miniwdl-plugins/archive/ad47c89d5152d6864f158520810fb2b4fa7eca24.zip#subdirectory=s3upload
```

Comment:
Wondering when would be a good time to just stuff miniwdl_plugins directly into this repo.